<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1600</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1600</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OFDR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="PlotWidget" name="OFDR_plot" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>1024</width>
      <height>768</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearpushButton">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="LogscalecheckBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string>Log scale</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="filenamelabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>820</y>
      <width>1021</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="ReflectancepeaksearchgroupBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>408</y>
      <width>501</width>
      <height>411</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="title">
     <string>Reflectance peak search</string>
    </property>
    <widget class="QTableWidget" name="LocalmaximumtableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>481</width>
       <height>311</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>230</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Distance (mm)</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Reflectance (dB)</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="FindpeakpushButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>24</y>
       <width>91</width>
       <height>58</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>Find peak</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="DistancerangesettinggroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>381</width>
       <height>61</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="title">
      <string>Distance range setting</string>
     </property>
     <widget class="QLabel" name="DistanceFromlabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>25</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Blank means search from 0 mm&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>From (mm)</string>
      </property>
     </widget>
     <widget class="QLabel" name="DistanceTolabel">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>25</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Blank means search to 5000 mm (maximum).&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>To (mm)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="DistanceFromlineEdit">
      <property name="geometry">
       <rect>
        <x>83</x>
        <y>25</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="DistanceTolineEdit">
      <property name="geometry">
       <rect>
        <x>273</x>
        <y>25</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="RunControlgroupBox">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>50</y>
      <width>501</width>
      <height>351</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="title">
     <string>Run Control</string>
    </property>
    <widget class="QPlainTextEdit" name="RunMessageplainTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>481</width>
       <height>241</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="RunMessagelabel">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>80</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="text">
      <string>Message</string>
     </property>
    </widget>
    <widget class="QPushButton" name="AutoMeasurepushButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>Auto Measure</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>1400</x>
      <y>820</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionquit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen_data_file">
   <property name="text">
    <string>Open data file (*.csv)</string>
   </property>
  </action>
  <action name="actionquit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionTLS_8164Awindow">
   <property name="text">
    <string>TLS 8164A</string>
   </property>
  </action>
  <action name="actionRazor_express">
   <property name="text">
    <string>Razor express</string>
   </property>
  </action>
  <action name="actionSave_OFDR_data_file">
   <property name="text">
    <string>Save OFDR data file</string>
   </property>
  </action>
  <action name="actionOpen_OFDR_data_file">
   <property name="text">
    <string>Open OFDR data file</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>AutoMeasurepushButton</tabstop>
  <tabstop>DistanceFromlineEdit</tabstop>
  <tabstop>DistanceTolineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>ClearpushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ClearButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LogscalecheckBox</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>plot_processed_data()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>81</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>92</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AutoMeasurepushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>AutoMeasureButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1554</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>1595</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>testButtonClicked()</slot>
  <slot>Open_data_file()</slot>
  <slot>ClearButtonClicked()</slot>
  <slot>test2ButtonClicked()</slot>
  <slot>plot_processed_data()</slot>
  <slot>FindpeakButtonClicked()</slot>
  <slot>AutoMeasureButtonClicked()</slot>
  <slot>Connection_TLS_8164_Clicked()</slot>
  <slot>Connection_Razor_express_Clicked()</slot>
  <slot>Save_OFDR_data_file()</slot>
  <slot>Open_OFDR_data_file()</slot>
 </slots>
</ui>
